### PR TITLE
Changed the family name default to be an empty string.

### PR DESCRIPTION
**Pull request by afzaledx**
LEARNER-1701

### CODE REVIEW

**Comment by afzaledx** [comment]
@edx/helio

**Review by mjfrey — APPROVED** [review]

File: course_discovery/apps/course_metadata/models.py [hunk 1]
```diff
@@ -222,7 +222,7 @@ def __str__(self):
 
     @property
     def full_name(self):
-        return ' '.join((self.given_name, self.family_name,))
+        return ' '.join((self.given_name, self.family_name,)) if self.family_name else self.given_name
```
Comment (clintonb) [inline course_discovery/apps/course_metadata/models.py]: Don't do this. Set the default value of `family_name` to an empty string and make the field non-nullable.

**Review by clintonb — CHANGES_REQUESTED** [review]

**Comment by clintonb** [comment]
See the note on commit messages at https://openedx.atlassian.net/wiki/display/ECOM/Developer+Onboarding. Shorten your commit summaries.

**Comment by afzaledx** [comment]
@clintonb , can you please review again?

**Comment by clintonb** [comment]
Let's discuss this on the ticket. Fixing this correctly will involve a bit more work, and I'm not convinced this is a real issue we have with our current data.

**Comment by clintonb** [comment]
This is not a priority. We'll get to it...eventually.

**Pull request closed without merging**